apps/web/src/app/admin/check-in/page.tsx

```diff
@@ -1,83 +1,64 @@
 import CheckinScanner from "@/components/admin/scanner/CheckinScanner";
-import FullScreenMessage from "@/components/shared/FullScreenMessage";
 import { db } from "db";
-import { eq, and } from "db/drizzle";
-import { events, users, scans } from "db/schema";
+import { eq } from "db/drizzle";
+import { userCommonData } from "db/schema";
 
 export default async function Page({
 	searchParams,
 }: {
 	searchParams: { [key: string]: string | undefined };
 }) {
-	// TODO: maybe move event existant check into a layout so it holds state?
```

Comment:
nit: I believe I saw you also do this somewhere else in the files, but be careful of removing old TODOs without documenting them somewhere or creating them as issues so thinks do not slip through the cracks.